code/network/multiui.cpp

```diff
@@ -4976,7 +4976,7 @@ void multi_create_accept_hit()
 								XSTR("A time limit is being used in a co-op game.\r\n"
 									"  Select \'Cancel\' to go back to the mission select screen.\r\n"
 									"  Select \'Yes\' to continue with this time limit.\r\n"
-									"  Select \'No\' to continue without this time limit.", -1));
+									"  Select \'No\' to continue without this time limit.", 1620));
```

Comment:
You have reused the id `1620` from another XSTR but that would likely not work in another language with retail since FSO would then use the wrong translated string. You need to make sure that all IDs you are using are new to avoid breaking retail or other translation mods.

Reply:
I am sorry. What you mean exactly? What have this change to do with retail? Is this ID already used in the Retail code aswell? 😮  I thought Retail is using IDs only up to 1589, because that is the last entry, that is used in the retail strings.tbl for all languages. Then i will go through the retail code aswell to check that.

Reply:
The problem I see here is that the string `"You have selected TCP/IP for multiplayer ..."` from `multi.cpp` is assigned to the XSTR ID 1620 at the moment. That might not be a valid ID for retail but a mod may have used it for translating this new message.

You assigned this message the new ID 1604 and then  used 1620 for the message `"A time limit is being used in a co-op ..."`. So now if a mod has translated what used to be ID 1620 based on the old string (`"You have selected TCP/IP for multiplayer ..."`), then the translated string would be used here where you now use the ID 1620.

My point is that you should not change the ID of existing `XSTR` calls (except if they refer to a duplicate entry then it's obviously OK) to avoid breaking the translation of old mods.

Reply:
That is the point, this 1620 was a duplicate. Line 338 in the mainhallmenu.cpp is using the same string with ID 1604.

And yes, i thought about the compatibility to other translation mods. But actually i was not able to find any, that is using more than the Retail 1589. Actually i am very certain, that there is any translation mod, that is using the additional strings by the SCP.

Reply:
If there are no mods that translate this ID (and you are right, there are practically no translation mods) then reassigning this ID wouldn't matter.

If we actually find a mod that translates this then either that mod could be fixed or the ID could be changed back to make it work again.